src/devices/DCMotor/DCMotorSettings.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Pwm;
+using System.Device.Pwm.Drivers;
+
+namespace Iot.Device.DCMotor
+{
+     /// <summary>
+     /// Settings for DCMotor class.
+     /// </summary>
+    public class DCMotorSettings
+    {
+        public int? Pin0 { get; set; }
```

Comment:
Should Pin0 and Pin1 have more descriptive names? Like what are they connected to for example, or what should each control? In other words, how do I know if one of the pins I'm using is Pin0 or Pin1? Having doc comments on the properties would help with this.

Reply:
The H-bridge you connect DC motor has 3 pins
- enable where you connect PWM
- 2x data

optionally you can not use enable pin (or always enable) and use 2x data pins - that may cause the H-bridge to get a bit warm for larger engines.

I find this settings a bit confusing as well but it is as clean as I could come up with until https://github.com/dotnet/iot/issues/204 is addressed. If this was fixed the way I proposed I could remove half of the options here and make this much more readable.

Swapping data pins will swap the rotation direction and there is not really a good name for that (depending which way the current flows and how much electrons flow you get different direction and speed). Currently it will work regardless of how you connect it (at least that was the intention).

I'll make sure to document this in the sample/README but don't have ideas for better name.